jaxb-ri/xjc/pom.xml

```diff
@@ -129,14 +129,26 @@
                         </configuration>
                     </execution>
                     <execution>
-                        <id>copy-lib</id>
+                        <id>copy-resolver</id>
                         <phase>validate</phase>
                         <goals>
                             <goal>copy-dependencies</goal>
                         </goals>
                         <configuration>
                             <outputDirectory>${project.build.directory}/mods</outputDirectory>
-                            <excludeArtifactIds>jmockit</excludeArtifactIds>
+                            <includeArtifactIds>resolver</includeArtifactIds>
```

Comment:
is resolver still needed these days? Isn't it included in JDK already?

Reply:
It exists in 8 and is removed in 11 because of new API. We use new API on 9+ and old packaged in JDK on 8, but need to compile against something classes dependent on 8's API with 11.